src/app/listingInfo/bio-blurb-info/bio-blurb-info.component.html

```diff
@@ -0,0 +1 @@
+<p>bio-blurb-info works!</p>
```

Comment:
whenever you get further along, you may want to use my bio shortener function to add the readmore/learn more sections as needed